.maestro/duckplayer/common/input_search.yaml

```diff
@@ -0,0 +1,7 @@
+appId: com.duckduckgo.mobile.android
+---
+# Select a YouTube video from SERP
+- tapOn:
+    id: "omnibarTextInput"
+- inputText: "https://duckduckgo.com/?q=DuckDuckGo%20vs%20Google%3A%205%20Reasons%20You%20Should%20Switch%20video%20site%3Ayoutube.com&ko=-1&ia=web"
```

Comment:
i feel like there should be a way to speed this part of the test up as inputting such a long string char-by-char is a bit slow, perhaps using the clipboard to paste it instead of typing it. I couldn't find a way even though I feel like it should be possible with the right command combo. 

I have asked on Maestro's slack to see but don't block on this; if there is a better way to do it can include in a follow-up.